python/felis/tap.py

```diff
@@ -34,14 +34,13 @@
 from sqlalchemy.schema import MetaData
 from sqlalchemy.sql.expression import Insert, insert
 
-from .check import FelisValidator
-from .types import FelisType
-from .visitor import Visitor
+import felis.datamodel as dm
```

Comment:
Better to avoid absolute imports if you can, `import .datamodel as dm` should work fine here. I'm not a big fan of very short identifiers, I'd probably use `from . import datamodel`.

Reply:
I think you're right that it would be clearer to use the module name rather than shorten it. I will update this.